### DISCUSSION TITLE
Display class

### DISCUSSION THREAD

**Discussion by tawonat7** [LWJGL/lwjgl3#705] [Q&A] [answered]
I wonder where the
`org.lwjgl.opengl.Display ` 
class went I can't acccess it anymore and I am now having trouble using the library

**Comment by octylFractal** [comment, marked as answer]
That is from LWJGL 2. You should use the GLFW bindings instead. Resources like [this](https://github.com/LWJGL/lwjgl3-wiki/wiki/2.6.6-LWJGL3-migration) may be useful.
